GetStream/stream-chat-swift
# [CIS-1782] Fix issue with duplicated giphy commands.

### 🔗 Issue Links

CIS-1782

### 🎯 Goal

Fix issue with duplicated giphy commands.

### 🎨 Showcase

| Before | After |
| ------ | ----- |
|  ![IMG_0171](https://user-images.githubusercontent.com/4924709/165077727-573353a2-465e-4a7b-bb4a-3bee7cc83004.PNG)   |  ![IMG_0322](https://user-images.githubusercontent.com/4924709/165077787-1748e4e8-1c8d-4239-8c42-6d2b221b336a.PNG)  |

### 🧪 Manual Testing Notes

⚠️ Issue only appears on device.

1. Create a conversation with a participant
2. Receive message from the participant
3. Verify the giphy commands are not duplicated.

### ☑️ Contributor Checklist

### 🎁 Meme

_Provide a funny gif or image that relates to your work on this pull request. (Optional)_
